lib/src/main/java/ECSE428Project/controller/LogoutController.java

```diff
@@ -0,0 +1,37 @@
+package ECSE428Project.controller;
+
+import ECSE428Project.dto.AccountDto;
+import ECSE428Project.service.LogoutService;
+//import ECSE428Project.controller.LoginController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@CrossOrigin(origins = "*")
+@RestController
+public class LogoutController {
+
+    @Autowired
+    private LogoutService logoutService;
+
+    //Logging out
+	@PostMapping(value = { "/logout", "/logout/" })
+	public AccountDto logoutProfile(@RequestParam("email") String email) {
+		if (!logoutService.profileLogout(email).isLoggedIn()) {
```

Comment:
I don't really understand, when do you actually log out the account?
I feel like you are just checking that it is logged in, but I could be misunderstanding your code.

Reply:
The controller calls "logoutProfile", which is the logout service method. The logout happens in there, and is saved in the database. Some other things could be done too after that when we have the menus figured out

Reply:
the check that happens in the controller is actually to make sure the user was logged out in the service call

Reply:
correction, the controller call "profileLogout". My mistake